Automattic/wp-calypso
# Hosting onboarding: Add shimmer loading animation to sites with AT transfer in progress on Sites page

Closes https://github.com/Automattic/dotcom-forge/issues/2376

## Proposed Changes
This PR adds a loading banner to an atomic transfer site to show that the transfer is in progress.
* Add loader notice that shows when site is transferring and when the transfer is completed.
* Add hook to check atomic transfer status. 

## Testing Instructions

1. Go to  `/sites` page.
2. get siteId of an atomic site
3. using  react dev tools select the top most component then right click the `store` and click `set as global variable`
4.  In the console run `$reactTemp0.dispatch({type: 'AUTOMATED_TRANSFER_STATUS_SET', siteId:atomicId, status: 'active' });` `$reactTemp` is what ever global variable the store is set to.
6.  Verify that the loading notice is shown when `active`.
7.  Verify `site activated notice` is shown when transfer completes (`it will fetch complete from server for exisiting AT site`)



* 
![image](https://github.com/Automattic/wp-calypso/assets/47489215/f331186f-f1e1-4456-9c5a-7ec0f1f80364)
![image](https://github.com/Automattic/wp-calypso/assets/47489215/a73d8982-83c0-4c1c-b724-5db19b340661)






## Pre-merge Checklist
